app/services/document_unpublishing_service.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class DocumentUnpublishingService
+  def retire(document, explanatory_note, locale: "en")
```

Comment:
If we're passing in document we can use that as the source of the locale rather than an extra argument

Reply:
Good point!